src/styles/blocks/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url('../../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
+}
```

Comment:
Avoid using tag names for styling as per the checklist. Since `.stars__star` is already a class, there's no need to specify the tag name. It's better to style using only the class to ensure that the styles are not overly specific and can be reused on different elements if needed.